src/lib.rs

```diff
@@ -172,6 +172,57 @@ struct Wallet {
     wallet_mutex: Mutex<BdkWallet<AnyDatabase>>,
 }
 
+pub struct OutPoint {
+    txid: String,
+    vout: u32,
+}
+
+pub struct TxOut {
+    value: u64,
+    address: String,
```

Comment:
At first glance it looks good to me, but it's still something I want to bring to the attention of other reviewers (@notmandatory, @artfuldev). The `TxOut` in BDK contains a `Script` instead of an address of type `String`. I don't think we need the `Script` at the moment, but it does change the shape of the API, so something to note.